src/mixins/object_interactivity.mixin.js

```diff
@@ -299,6 +299,17 @@
      */
     onSelect: function() {
       // implemented by sub-classes, as needed.
-    }
+    },
+
```

Comment:
I guess we should/could add `onDragStart: () => false` here, as `onDragStart` and `canDrop` are public methods to for overriding